src/test/java/bio/terra/service/dataset/DatasetDaoTest.java

```diff
@@ -525,11 +525,18 @@ protected void assertTablesInRelationship(Dataset dataset) {
   }
 
   protected void assertDatasetTable(Table table) {
-    if (table.getName().equals("participant")) {
-      assertThat("participant table has 4 columns", table.getColumns(), hasSize(4));
-    } else {
-      assertThat("other table created is sample", table.getName(), equalTo("sample"));
-      assertThat("sample table has 3 columns", table.getColumns(), hasSize(3));
+    switch (table.getName()) {
+      case "participant":
+        assertThat("participant table has 4 columns", table.getColumns(), hasSize(4));
+        break;
+      case "sample":
+        assertThat("sample table has 3 columns", table.getColumns(), hasSize(3));
+        break;
+      case "123_leading_number":
+        assertThat("123_leading_number table has 1 column", table.getColumns(), hasSize(1));
+        break;
+      default:
+        throw new RuntimeException("Unexpected table " + table.getName());
     }
```

Comment:
If you use the enhanced switch form you can avoid having to write `break`:
```suggestion
    switch (table.getName()) {
      case "participant" ->
          assertThat("participant table has 4 columns", table.getColumns(), hasSize(4));
      case "sample" -> assertThat("sample table has 3 columns", table.getColumns(), hasSize(3));
      case "123_leading_number" ->
          assertThat("123_leading_number table has 1 column", table.getColumns(), hasSize(1));
      default -> throw new RuntimeException("Unexpected table " + table.getName());
    }
```

You can enable a java inspection in IntelliJ to flag and automatically rewrite `switch` statements into this form. The inspection is called `Statement can be replaced with enhanced 'switch'`.